public/plugin.tsx

```diff
@@ -146,12 +160,18 @@ export class AssistantPlugin
         navLinkStatus: AppNavLinkStatus.hidden,
         mount: async (params: AppMountParameters) => {
           const [coreStart, pluginsStart] = await core.getStartServices();
-          const { renderText2VizApp } = await import('./text2viz');
-          return renderText2VizApp(params, {
-            ...coreStart,
-            ...pluginsStart,
-            setHeaderActionMenu: params.setHeaderActionMenu,
-          });
+          const isSubscriptionActive =
+            coreStart.application.capabilities?.assistant?.isSubscriptionActive === true;
+          if (isSubscriptionActive) {
+            const { renderText2VizApp } = await import('./text2viz');
+            return renderText2VizApp(params, {
+              ...coreStart,
+              ...pluginsStart,
+              setHeaderActionMenu: params.setHeaderActionMenu,
+            });
+          } else {
+            return () => {};
+          }
```

Comment:
do we want to render some tips or instructions to direct user to active the subscription?

Reply:
Add a default "Application Not Activated" danger tips now